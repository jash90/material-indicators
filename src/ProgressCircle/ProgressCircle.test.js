import React from 'react';
import ProgressCircle from './ProgressCircle.js';

import renderer from 'react-test-renderer';

test('ProgressCircle Renders', () => {
  const tree = renderer.create(<ProgressCircle />).toJSON();
  expect(tree).toMatchSnapshot();
});
