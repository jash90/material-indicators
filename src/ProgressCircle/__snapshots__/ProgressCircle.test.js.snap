// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressCircle Renders 1`] = `
<View>
  <View
    style={
      Object {
        "height": 48,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 48,
      }
    }
  >
    <View
      collapsable={false}
      style={
        Object {
          "height": 24,
          "overflow": "hidden",
          "width": 48,
        }
      }
    >
      <View
        style={
          Object {
            "height": 48,
            "transform": Array [
              Object {
                "rotate": "-165deg",
              },
            ],
            "width": 48,
          }
        }
      >
        <View
          collapsable={false}
          style={
            Object {
              "height": 24,
              "overflow": "hidden",
              "width": 48,
            }
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgb(0, 0, 0)",
                "borderRadius": 24,
                "borderWidth": 4.8,
                "height": 48,
                "width": 48,
              }
            }
          />
        </View>
      </View>
    </View>
  </View>
</View>
`;
